src/util/util.c

```diff
@@ -622,7 +622,7 @@ bool check_ipv4_addr(struct in_addr *addr, uint8_t flags)
         DEBUG(SSSDBG_FUNC_DATA, "Multicast IPv4 address %s\n", straddr);
         return false;
     } else if ((flags & SSS_NO_LOOPBACK)
-               && inet_netof(*addr) == IN_LOOPBACKNET) {
+               && ((ntohl(addr->s_addr) & 0xff000000) == 0x7f000000)) {  /* from <linux/in.h> */
```

Comment:
Wouldn't it be better to use the `IN_LOOPBACK` macro directly?

Reply:
In the past we received ticket where code base failed to build with `musl` libc.

Reply:
Makes sense.

Reply:
Hi,

I still like the idea of the macro here. What about
```
#ifndef IN_LOOPBACK
#define IN_LOOPBACK .....
#endif
```
or defining a `SSS_IN_LOOPBACK`?

bye,
Sumit

Reply:
ok, updated.